src/hacbs/components/Commits/CommitDetailsView.tsx

```diff
@@ -0,0 +1,197 @@
+          description={
+            <>
+              <Text component="p" className="pf-u-mt-lg pf-u-mb-xs">
+                <span className="pf-u-mr-sm">Commit ID:</span>
+                <ExternalLink href={commit.shaURL}>
+                  {commitName}
+                  <span className="pf-u-mr-sm pf-u-ml-sm">
+                    <GithubIcon />
+                  </span>
+                </ExternalLink>
+                <StatusIconWithText status={status} />
+              </Text>
+              <Text component="p" className="pf-u-mt-xs pf-u-mb-xs">
+                Branch:{' '}
+                {commit.branch.length > 0
+                  ? commit.branch.map((branch, index) => {
+                      return (
+                        <>
+                          {commit.gitProvider === 'github' && commit.repoOrg ? (
+                            <ExternalLink
+                              href={`https://github.com/${commit.repoOrg}/${commit.repoURL}/tree/${branch}`}
+                              text={`${branch}`}
+                            />
+                          ) : (
+                            `${branch}`
+                          )}
+                          {index < commit.branch.length - 1 && `,`}
+                        </>
+                      );
+                    })
+                  : '-'}
+              </Text>
+              <Text component="p" className="pf-u-mt-xs pf-u-mb-xs">
+                <span className="pf-u-color-200">
+                  By {commit.user} at <Timestamp timestamp={commit.creationTime} />
+                </span>
+              </Text>
+              {commit.shaTitle && <p className="pf-u-mt-xs pf-u-mb-xs">{`"${commit.shaTitle}"`}</p>}
+              <Text component="p" className="pf-u-mt-sm pf-u-mb-sm">
+                Component:{' '}
+                {commit.components.map((component, index) => {
+                  return (
+                    <>
+                      {component}
+                      {index < commit.components.length - 1 && ','}
+                    </>
+                  );
+                })}
+              </Text>
+            </>
+          }
+          actions={[
+            {
+              key: 'go-to-source',
+              label: 'go to source',
+              onClick: () => {
+                navigate(`/app-studio/import?application=${applicationName}`);
+              },
+            },
+          ]}
+          tabs={[
+            {
+              key: 'overview',
+              label: 'Overview',
+              isFilled: true,
+              component: <CommitsOverviewTab commit={commit} />,
+            },
+            {
+              key: 'pipelineruns',
+              label: 'Pipeline runs',
+              component: <CommitsPLRTab commit={commit} />,
+            },
+            {
+              key: 'events',
+              label: 'Events',
+              component: <CommitsPLRTab commit={commit} />,
+            },
```

Comment:
You may as well stub out proper components for each tab. Or otherwise just remove the tab for now.